### PR TITLE
Update FastlaneMatch.md

### PR DESCRIPTION
**Pull request by ilyapuchka**
Make commands more clear

### CODE REVIEW

File: Cookbook/Technical-Documents/FastlaneMatch.md [hunk 1]
```diff
@@ -55,14 +55,26 @@ We specify individual app identifiers instead of adding all App Store bundle ids
...
 (As of June 2019, `fastlane` could not renew expired profiles automatically or using a command although the documentation mentions that it does)
-- Renew the profile on Apple developer center
+
+- Renew the profile on Apple developer center:
+  - go to "Edit" profile
+  - save it without changing anything
+  
 - Run the following command to update match repository with new profile
 
+To update enterprise profile:
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion
To update the enterprise profile:
```

File: Cookbook/Technical-Documents/FastlaneMatch.md [hunk 2]
```diff
@@ -55,14 +55,26 @@ We specify individual app identifiers instead of adding all App Store bundle ids
...
+
+```shell
+ $ bundle exec fastlane match enterprise --team_id {{enterprise teamID}} --app_identifier {{enterprise bundleID}} --readonly false
+```
+
```
Comment (diegopetrucci) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion

E.g:
```shell
  $ bundle exec fastlane match enterprise --team_id 12345 --app_identifier com.babylon.yourApp --readonly false
 ```
```

**Review by diegopetrucci — APPROVED** [review]

**Review by zzcgumn — APPROVED** [review]
I agree that this makes it easier to understand.

File: Cookbook/Technical-Documents/FastlaneMatch.md [hunk 3]
```diff
@@ -55,14 +55,26 @@ We specify individual app identifiers instead of adding all App Store bundle ids
...
+To update appstore profile:
```
Comment (ilyapuchka) [inline Cookbook/Technical-Documents/FastlaneMatch.md]:
```suggestion
To update the AppStore profile:
```

**Pull request merged**